rocketpy/mathutils/function.py

```diff
@@ -1087,6 +1087,36 @@ def to_frequency_domain(self, lower, upper, sampling_frequency, remove_dc=True):
             extrapolation="zero",
         )
 
+    def low_pass_filter(self, alpha):
+        """
+        Implements a low pass filter with a moving average filter
+
+        Parameters:
+        - alpha (float): attenuation coefficient, 0 < alpha < 1
+
+        - for a given dataset, the larger alpha is, the more closely the filtered function returned will match the function
```

Comment:
The docstring is passing the column number 80, this is something not illegal but not desirable as well. 

Also, rocketpy follows the numpy standard, so I'd ask you to read other methods docstring to see how to format this, but basically you're going to need to add "---------" under the "Parameters".

Reply:
This have been changed by ClementMonaco : 
https://github.com/RocketPy-Team/RocketPy/pull/508/commits/d8dce0cd775efedc3d16b46f6f3f5c42df8636af

Reply:
The documentation with sphinx is really sensitive, so I had to modify the docstring in the last commit ([2085990](https://github.com/RocketPy-Team/RocketPy/pull/508/commits/2085990e73f53832ac9e8b5062c538ae14da0abb)) so the docs get rendered correctly.

Before: 
![image](https://github.com/RocketPy-Team/RocketPy/assets/63590233/c2f498df-8d72-4a54-ae74-2481d98d95ef)


After: 
![image](https://github.com/RocketPy-Team/RocketPy/assets/63590233/6c09fa78-b05f-48f0-93e0-0f0aafc5cf66)